backend/app/tests/test_stripe_billing.py

```
@@ -0,0 +1,237 @@
+def _stripe_settings(**overrides):
+    data = {
+        "STRIPE_SECRET_KEY": "sk_test_123",
+        "STRIPE_WEBHOOK_SECRET": "whsec_test",
+        "STRIPE_PRICE_PLAN_MAP": json.dumps({"price_starter": STARTER_PLAN_CODE}),
+        "STRIPE_API_BASE_URL": "https://api.stripe.test/v1",
+    }
+    data.update(overrides)
+    return SimpleNamespace(**data)
+
+
+def _stripe_signature(payload: dict, secret: str = "whsec_test") -> tuple[bytes, str]:
```

Ruff (0.15.18)

[error] Possible hardcoded password assigned to function default: "secret"

(S107)